src/common/clipboard.c

```diff
@@ -154,13 +155,20 @@ void guac_common_clipboard_append(guac_common_clipboard* clipboard, const char*
     if (remaining < length)
         length = remaining;
 
+    char output_data[length];
+    memset(output_data, '\0', length);
+    char* output = output_data;
+
+    /* Convert clipboard contents */
+    guac_iconv(GUAC_READ_UTF8_NORMALIZED, &data, length,
+               GUAC_WRITE_UTF8, &output, length);
+
     /* Append to buffer */
-    memcpy(clipboard->buffer + clipboard->length, data, length);
+    memcpy(clipboard->buffer + clipboard->length, output_data, length);
```

Comment:
Changing this here within `guac_common_clipboard` will affect all current protocols, including VNC and RDP. Is this intended?

Reply:
Ahh that's true, moved to terminal